training/src/main/java/wolox/training/models/Book.java

```diff
@@ -39,8 +42,17 @@
     @Column(nullable = false)
     private String isbn;
 
-    Book(String genre, String author, String image, String title, String subtitle, String publisher,
+
+    @ManyToMany(mappedBy = "books")
+    private List<Users> users;
+
+    public Book() {
+    }
+
+    public Book(String genre, String author, String image, String title, String subtitle,
+        String publisher,
         String year, Integer pages, String isbn) {
+        List<Users> users = null;
```

Comment:
You could initialize it as a `new ArrayList()`. It wouldn't be necessary to assign an empty list after doing this.